test/helpers/core.js

```diff
@@ -11,5 +11,6 @@ export default function createCore() {
     relayServiceEndpoint: process.env.RELAY_SERVICE_ENDPOINT,
     safeMasterAddress: process.env.SAFE_ADDRESS,
     subgraphName: process.env.SUBGRAPH_NAME,
+    databaseSource: process.env.DATABASE_SOURCE,
```

Comment:
can you upload the env file with this already updated? Also is the idea to have only one source? I thought we were thinking on having 2/3 option one as fallback?

Reply:
The idea is configure the source option outside the core, and the core only reads the env var. The alternatives are 2 so far and will have a code like:
- `graph-node`
- `land-db`

Reply:
btw, the updated env file is: `[.env.example](https://github.com/CirclesUBI/circles-core/pull/105/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8c)`